lib/rules/no-tests-contain-correct.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        'Prevent the usage of the word "correct" in test statements.',
+      category: 'Best Practices',
+      recommended: false,
+      uri:
+        'https://github.com/Shopify/eslint-plugin-shopify/blob/master/docs/rules/no-tests-contain-correct.md',
+    },
+  },
+  create(context) {
+    const ignore = (context.options[0] && context.options[0].ignore) || [];
+    const ignoredFunctionNames = ignore.reduce((accumulator, value) => {
+      accumulator[value] = true;
+      return accumulator;
+    }, Object.create(null));
+
+    function isIgnoredFunctionName(node) {
+      return ignoredFunctionNames[node.callee.name];
+    }
+
+    return {
+      CallExpression(node) {
+        const erroneousMethod = descriptionContainsCorrect(node);
+
+        if (erroneousMethod && !isIgnoredFunctionName(node)) {
```

Comment:
Seems like a weird flow to me — I'd expect it to check if this is one of the functions we expect, removing ignored ones, then check if the description contains invalid words, then report.